### ISSUE TITLE
Sass source map not working

### ISSUE THREAD

**Issue by pakit84** [driftyco/ionic-app-scripts#926]
Hello ,
I am not able to generate scss source maps.
After "ionic serve" I'm navigating to www/build folder, there is main.css.map but it contains null instead of generated sourcemaps.

Do you have any idea why this is happening?

**Steps to reproduce:**
1. app-scripts 1.3.0 or higher 
2. ionic serve
3. go to www/build and check main.css.map : shoul contain "null" string

```
insert any relevant code between the above and below backticks
```

**Which @ionic/app-scripts version are you using?**

1.3.0, issue even with the latest version

Thanks.

Regards.

**Comment by danbucholtz** [comment]
To be totally honest, we probably aren't going to invest any resources into this anytime soon. There are too many higher priority items. We'll take a well-tested PR though.

Thanks,
Dan

**Comment by ElWardi** [comment]
Well, sass source map is disabled by default ,  (https://github.com/driftyco/ionic-app-scripts/blob/master/config/sass.config.js#L16) 

You can activate that manually by adding your own build script on `package.json`

```
{
//...
   "config": {
    "ionic_sass": "./scripts/sass.config.js"
  },
//...
}
```

**Comment by hikumealan** [comment]
Seems like a easy fix. Hopefully it can get into the next release.

**Comment by HugoHeneault** [comment]
@ElWardi What's inside the sass.config.js ? :)

**Comment by ElWardi** [comment]
@hikumealan 
I dont think this should be changed as source map should be available for development only. 
Maybe a documentation is required. 

@HugoHeneault  
A link to the sass.config.js is available on my above answer. I just copied the whole file as i am not sure if the file will be overridden or merged when building the config.

**Comment by lucasprogamer** [comment]
@HugoHeneault  work's, thks
